### PR TITLE
Setup hermetic build, with option for remote

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

File: pkg/arcs-import.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const locale = globalThis.config?.localArcsjs ? 'local': 'cdn';
```
Comment (shaper) [inline pkg/arcs-import.js:8]: Needs header.

File: pkg/env/arcs-cdn.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+// CDN
```
Comment (shaper) [inline pkg/env/arcs-cdn.js:8]: Needs header (and next couple of files too).

File: pkg/env/arcsjs-apps/pkg/Library/Common/dom/surfaces/remote/smoke.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<!doctype html>
```
Comment (shaper) [inline pkg/env/arcsjs-apps/pkg/Library/Common/dom/surfaces/remote/smoke.html]: Needs header.

File: pkg/env/arcsjs-apps/pkg/Library/Common/dom/surfaces/xen/surface-walker.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
+import {Xen} from '../../xen/xen-async.js';
```
Comment (shaper) [inline pkg/env/arcsjs-apps/pkg/Library/Common/dom/surfaces/xen/surface-walker.js:8]: Needs header.

File: pkg/env/arcsjs-apps/pkg/Library/Common/dom/xen/rollup.config.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import {terser} from 'rollup-plugin-terser';
```
Comment (shaper) [inline pkg/env/arcsjs-apps/pkg/Library/Common/dom/xen/rollup.config.js:7]: Needs header.

File: pkg/env/arcsjs-apps/pkg/arcs-import.js [hunk 6]
```diff
@@ -0,0 +1 @@
+export * from './env/arcs-local.js';
```
Comment (shaper) [inline pkg/env/arcsjs-apps/pkg/arcs-import.js:8]: Needs header here and next couple of files.

**Review by shaper — APPROVED** [review]
There are a few icons (material-xen, and shell assets//users), do we need these?

File: pkg/env/arcsjs-apps/pkg/Library/Common/shell/services.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+        return {
+          runtime
+        };
+      }
+    }
+  },
+  user: async (runtime, host, request) => {
+    switch (request.msg) {
+      case 'particle-error': {
+        //console.error(error);
```
Comment (shaper) [inline pkg/env/arcsjs-apps/pkg/Library/Common/shell/services.js]: Maybe this is worth uncommenting? Or perhaps this is just a template? I have not looked at usage yet.

File: pkg/env/arcsjs-apps/pkg/arcs-import.js (hunk 6 above)
Comment (sjmiles) [reply pkg/env/arcsjs-apps/pkg/arcs-import.js:8]: There is no sane universe in which `export * from path` is copyrightable, but ok. 

Each time we are forced to do this, we add mass to the _unbundled_ version, thus promulgating the cargo-cult of bundling and transpiling. 

This is tail wagging dog.

**Comment by shaper** [comment]
I'm assuming this is ready to go, but if you had more planned... let me know.

**Pull request merged**